### PR TITLE
Bring upload state from file input into view

### PR DESCRIPTION
**Pull request by benalleng**
As mentioned in issue #192 the upload state from file was not rendering. Adding a label and hiding the default input file upload boilerplate I styled it to look like the buttons above it.

### CODE REVIEW

**Review by benthecarman — APPROVED** [review]
Works for me! Would wait on @futurepaul's approval tho

**Review by futurepaul — APPROVED** [review]
wonderful, thank you!

**Pull request merged**
